Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1251,6 +1267,48 @@ internal IEnumerable<Block<T>> IterateBlocks(int offset = 0, int? limit = null)
             }
         }
 
+        private Block<T> MineGenesis(
```

Comment:
How about using `IBlockPolicy` to define an action for the genesis block(i.e. `IBlockPolicy.GensisAction[s]`) instead of `MineGenesis()`? if so, we can guide to treat genesis as only data, not sort of code.

Reply:
Oh I'm too late, sorry :disappointed: 
It looks like good idea, but if we applied it, the hash of genesis-block won't be affected from `actions` in genesis-block. In my thought, then there is no way to know distinguish between different chains.